packages/language-server/src/plugins/html/features/astro-attributes.ts

```diff
@@ -0,0 +1,130 @@
+import { newHTMLDataProvider } from 'vscode-html-languageservice';
+
+export const astroAttributes = newHTMLDataProvider('astro-attributes', {
+	version: 1,
+	globalAttributes: [
+		{
+			name: 'class:list',
+			description: 'Utility to provide a list of class',
+		},
+		{
+			name: 'set:html',
+			description: 'Inject unescaped HTML into this tag',
+			references: [
+				{
+					name: 'Astro documentation',
+					url: 'https://docs.astro.build/en/migrate/#deprecated-unescaped-html',
```

Comment:
Yes! Love that you can link to the docs this way.

Reply:
Yep! It's really neat. It's unfortunate that there's no way to provide this for diagnostics though, it only works for hover info